pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/base/PensjonLatex.kt

```diff
@@ -415,11 +429,73 @@ object PensjonLatex : BaseTemplate() {
 
             is Element.NewLine ->
                 printWriter.printCmd("newline")
+            is Element.Table ->
+                with(printWriter) {
+                    val cellWidths = element.rows.map { it.cells.sumOf { cell -> cell.cellColumns } }.distinct()
+                    verifyCells(cellWidths)
```

Comment:
Jeg foreslår at i stedenfor at vi tar denne valideringen når et brev blir bestilt for malen, så gjør vi det som assertions på Element.Table i en `init` blokk. På den måten vil brevbaker feile umiddelbart, og du kan lagre bredden som en variabel på Table klassen.

Reply:
Tar gjerne mot litt hjelp for denne :) prøvde selv å gjøre de obligatoriske.